core/src/test/scala/org/apache/spark/scheduler/TaskSchedulerImplSuite.scala

```diff
@@ -1911,11 +1912,12 @@ class TaskSchedulerImplSuite extends SparkFunSuite with LocalSparkContext with B
     clock.advance(2000)
 
     // Now give it some resources and both tasks should be rerun
-    val taskDescriptions = taskScheduler.resourceOffers(IndexedSeq(
-      WorkerOffer("executor2", "host2", 1), WorkerOffer("executor3", "host3", 1))).flatten
-    assert(taskDescriptions.size === 2)
-    assert(taskDescriptions.map(_.index).sorted == Seq(0, 1))
-    assert(manager.copiesRunning.take(2) === Array(1, 1))
+    submitTasksAndCheck(IndexedSeq(WorkerOffer("executor2", "host2", 1),
+      WorkerOffer("executor3", "host3", 1)), Seq.empty) { taskDescriptions =>
```

Comment:
Personally, I feel it's weird that we're submitting empty task sets here... `submitTasksAndCheck` maybe not appropriate for this case? Or you could redesign the `submitTasksAndCheck`?

Reply:
OK. Let's not change this matter.